app/src/main/java/com/github/multimatum_team/multimatum/model/Deadline.kt

```diff
@@ -36,7 +36,9 @@ enum class DeadlineState {
 data class Deadline(
     val title: String,
     val state: DeadlineState,
-    val date: LocalDate
+    val date: LocalDate,
+    val description: String = "",
+    val id: Int = 0
```

Comment:
Firebase generates random IDs automatically, but they're represented as strings.
Please make it so that `id` is of type `String`.
Defining default values for the new fields is okay for now if you want to get things working quickly, but is ultimately going to cause us some problems, we'll have to remove the default values and fix code breakage in the next sprint.